client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts

```diff
@@ -18,7 +18,7 @@ const isIdValid = (id: string | null | undefined) => {
       sum += digitMul;
     })
   }
-  return sum % 10 === 0;
+  return sum % 10 === 0 && sum > 0;
```

Comment:
weird, that's a fix i already pushed to dev

Reply:
did you push it to test?

Reply:
yes